docs/source/engine.rst

```diff
@@ -0,0 +1,24 @@
+pyroengine.engine
+=================
+
+The models subpackage contains everuthing to manage the whole Fire Detection process by capturing and saving the image and
+by predicting if there is a fire or not based on this image
+
+
+.. currentmodule:: pyroengine.engine
+
+Pyronear Predictor
+------------------
+
+The pyronear_predictor class that loads the last pytonear model and makes a prediction on a given image.
```

Comment:
typo here : PyronearPreditor

Reply:
Actually , you can even remove the whole line considering the docstring will be displayed in the documentation